doc/nrf/includes/spm.txt

```diff
@@ -0,0 +1,4 @@
+:ref:`secure_partition_manager` must be programmed on the board.
+
+The sample is configured to compile and run as a non-secure application on nRF91's Cortex-M33.
```

Comment:
A bit confused here. Who is including this file?

Reply:
Ah, it is supposed to be included by any sample doc?

Reply:
Yes, exactly - the samples that need it.